recipes/itk/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.7)
+project(example)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+include_directories(${CONAN_INCLUDE_DIRS_ITK_SUB})
```

Comment:
You don't need to include the header for directly, conan_basic_setup should be able to detect the correct folder

Reply:
i do use the hook. Its in the guidelines

Reply:
There is a `CONAN_INCLUDE_DIRS_ITK` which is automatically managed. But for now, using `CONAN_INCLUDE_DIRS_ITK_SUB` manually is mandatory. Without it, it is impossible to `#include` any of the VNL headers included in ITK.